playbooks/roles/deploy-osh/defaults/main.yml

```diff
@@ -99,4 +99,4 @@ suse_osh_deploy_neutron_yaml_overrides: {}
 suse_osh_deploy_tempest_yaml_overrides: {}
 
 # override in your extravars to deploy and run tempest
-deploy_tempest: False
```

Comment:
would prefer if this was set to false+ ENV var so we can override it from the CI instead

Reply:
We have it true by default for airship so why not for osh?

Reply:
you are right here!